pkg/cmd/root/root.go

```diff
@@ -86,9 +86,9 @@ func NewRootCmd(f *cmdutil.Factory) *cobra.Command {
 
 	// API related commands
 	cmd.AddCommand(search.NewSearchCmd(f))
-	cmd.AddCommand(index.NewIndexCmd(f))
+	cmd.AddCommand(indices.NewIndicesCmd(f))
 	cmd.AddCommand(objects.NewObjectsCmd(f))
-	cmd.AddCommand(apikey.NewAPIKeyCmd(f))
+	cmd.AddCommand(apikeys.NewAPIKeysCmd(f))
```

Comment:
Can't make an edit suggestion, but check line 56, 57 in this file. They need to be updated as well.

Reply:
@kai687 Good catch 👍